lib/signs/realtime.ex

```diff
@@ -101,6 +103,16 @@ defmodule Signs.Realtime do
       text_zone: Map.fetch!(config, "text_zone"),
       audio_zones: Map.fetch!(config, "audio_zones"),
       source_config: source_config,
+      default_mode:
+        config
+        |> Map.get("default_mode")
+        |> then(fn
```

Comment:
nit: Since the need for this is so narrow, I don't think we need to exhaustively support all modes. I think we could just match `auto`, and have a default of `off`, and call it a day.

Reply:
I thought it might come in handy, if we wanted to mimic the effect of OIOs setting a particular mode in Signs UI, to support them all (since we cannot do this via Signs UI for the lab sign). Maybe I'm not quite understanding what the lab sign will be used for, though. If you're confident we won't need anything other than `auto`, then we can drop the rest.

Reply:
The main purpose of the test signs is the physical hardware itself, so we just need some kind of messages going to it. Any testing of the config logic we can easily do in dev with an existing sign and signs-ui, so all we need is a quick way to force auto mode here.